lib/locale_threads.t

```diff
@@ -1,30 +1,445 @@
 use strict;
+#23456789112345678921234567893123456789412345678951234567896123456789712345678981
```

Comment:
What is the purpose of the line above?

Reply:
It is a relic that I should have removed, and will do so before any push